sled-hardware/src/lib.rs

```diff
@@ -39,6 +40,20 @@ pub enum HardwareUpdate {
     DiskRemoved(UnparsedDisk),
 }
 
+/// Configuration for forcing a sled to run as a Scrimlet
+#[derive(Clone, Debug, Deserialize, Copy)]
+#[serde(rename_all = "snake_case")]
+pub enum ScrimletMode {
+    /// Force sled to run as a Gimlet
+    /// this is to preserve the old behavior of `scrimlet_override = false`,
+    /// but I haven't found where that logic has actually been leveraged...
```

Comment:
Not sure, but this comment might be outdated. I don't see any hits for the string `scrimlet_override` in the latest main. Maybe you mean `"stub_scrimlet"`?

Reply:
Yeah, I meant `stub_scrimlet`.